docs/UserGuide/API/Programming-JDBC.md

```diff
@@ -23,6 +23,10 @@
 
 ## JDBC
 
+*NOTICE: CURRENTLY, JDBC IS USED FOR CONNECTING SOME THIRD-PART TOOLS. 
+IT CAN NOT PROVIDE HIGH THROUGHPUT FOR WRITE OPERATIONS. 
+PLEASE USE [JAVA NATIVE API](Programming-Native-API.md) INSTEAD*
```

Comment:
Use the website url?
https://iotdb.apache.org/UserGuide/V0.12.x/API/Programming-Native-API.html